bot/bot.py

```diff
@@ -74,6 +74,8 @@ async def start(self, *args, **kwargs) -> None:
         """Async wrapper to load all the extensions."""
         from bot.utils.extensions import walk_extensions
 
+        await bot.unloads_cache.set("bot.exts.evergreen.xkcd", "https://dummy-msg-link.com")
+
```

Comment:
Can this be removed please, as a rule of thumb unless a PR is marked as a Draft or has requested changes you should feel comfortable having it merged at any time.

Reply:
Yes, I thought I removed it from my commit, was testing... so needed that.